src/Hazelcast.Net/DistributedObjects/IHCollection.cs

```diff
@@ -18,18 +18,13 @@
 
 namespace Hazelcast.DistributedObjects
 {
-    /// <summary>Concurrent, distributed, partitioned, listenable collection.</summary>
-    /// <remarks>Concurrent, distributed, partitioned, listenable collection.</remarks>
+    /// <summary>
+    /// Concurrent, distributed, partitioned, listenable collection.
```

Comment:
Since for instance `IHList` is not partitioned, maybe don't mention "partitioned" here. Also, we try to use "_Represents a ..._" for classes, and "_Defines a ..._" for interfaces, so here it would be "_Defines a concurrent, distributed collection._" or something equivalent.

Reply:
ok